applications/tari_app_grpc/proto/types.proto

```diff
@@ -228,3 +228,43 @@ message ConsensusConstants {
     /// Block weight for kernels
     uint64 block_weight_kernels = 16;
 }
+
+
+
+message Address{
+    bytes address =1;
+    string last_seen = 2;
+    uint32 connection_attempts = 3;
+    uint32 rejected_message_count = 4;
+    uint64 avg_latency = 5;
+}
+
+message Peer{
+    /// Public key of the peer
+    bytes public_key =1;
+    /// NodeId of the peer
+    string node_id =2;
+    /// Peer's addresses
+    repeated Address addresses = 3;
+    /// Last connection attempt to peer
+    string last_connection = 4;
+    /// Flags for the peer.
+    string flags = 5;
+    string banned_until= 6;
+    string banned_reason= 7;
+    string offline_at = 8;
+    /// Features supported by the peer
+    string features = 9;
+    /// Connection statics for the peer
+    string last_connected_at = 10;
```

Comment:
String seems like the wrong data type for times

Reply:
Its a date. Could perhaps convert this to a i64 so it represents a unix timestamp?

Reply:
protobuf `Timestamp` maybe?